airbyte-integrations/connectors/source-zendesk-talk/metadata.yaml

```diff
@@ -30,7 +30,12 @@ data:
       enabled: true
   releaseStage: generally_available
   supportLevel: certified
+  releases:
+    breakingChanges:
+      1.0.0:
+        upgradeDeadline: "2024-05-31"
+        message: "The source Zendesk Talk connector is being migrated from the Python CDK to our declarative low-code CDK. Due to changes to the incremental stream state message format and the removal of a nonexistent field from the ivrs stream schema, this migration constitutes a breaking change. After updating, please reset your source before resuming syncs. For more information, see our migration documentation for source Zendesk Talk."
```

Comment:
@katmarkham can you review the breaking change banner when you get a chance? The connector will be good to go once that's done